source/api/image/3.0/index.md

```diff
@@ -814,6 +814,8 @@ A recipe for setting this header on the Apache HTTP Server is shown in the [Apac
 
 ## 7. Server Responses
 
+Servers _MUST_ support the HTTP GET method for retrieval of Image API resources. Servers _SHOULD_ support the HTTP OPTIONS method as part of the [CORS][image30-cors-response] preflight request pattern, and it is _RECOMMENDED_ that implementations also support the HTTP HEAD method.
```

Comment:
Both SHOULD and RECOMMENDED (that mean the same thing) in the same sentence reads a little strange to me. Not a -1 but how about ...
> ... and SHOULD also support the HTTP HEAD method.

And then a little clause explaining why (e.g. to allow clients to test for the existence of the resource).

Reply:
Discussed, not really necessary.